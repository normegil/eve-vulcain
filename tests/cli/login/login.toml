bin.name = "eve-vulcain"
args = "login -vvv"
status.code = 0
