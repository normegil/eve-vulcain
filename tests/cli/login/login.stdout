Connected as EveVulcain Test User.
Refresh token: ABCDEFGHIJKLMN

You were already logged in.
To reconnect with another character, please call 'eve-vulcain logout' before trying to login again.
